### PR TITLE
Create Amber Network Documentation

### PR DESCRIPTION
**Pull request by Afauvre**
Initial draft of amber network documentation

### CODE REVIEW

File: src/operators/AmberNetwork [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+contingent on commission rates that are no more than +/- 10% of the 
+weighted median network commission rate)**
+
+## Wallet-Testers Needed
+
+More on this to come soon. 
+
+**Disclaimer: The Amber Network and all associated activities and rewards
+is not available to all participants and eligibility requirements apply.
+For details, see the Program Terms available here. This article does not
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:70]: Needs a link to program terms.

File: src/operators/AmberNetwork [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+as availability rankings and overall track record. **(NOTE: Delegation is
+contingent on commission rates that are no more than +/- 10% of the 
+weighted median network commission rate)**
+
+## Wallet-Testers Needed
+
+More on this to come soon. 
+
+**Disclaimer: The Amber Network and all associated activities and rewards
+is not available to all participants and eligibility requirements apply.
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:69]:
```suggestion
are not available to all participants and eligibility requirements apply.
```

File: src/operators/AmberNetwork [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# The Oasis Amber Network
+
+## What is it?
+With the Questnet reaching a point of stability, the Amber
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:4]:
```suggestion
With the [QuestNet](./the-quest-rules.md) reaching a point of stability, the Amber
```

File: src/operators/AmberNetwork [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+Network is the next phase of the Oasis Network’s path to Mainnet.
+The Oasis Amber Network is a release candidate for Mainnet and
+includes all the core features of the network you can expect
+at Mainnet launch.  This includes:
+ 
+* Fully decentralized consensus layer
+* Staking and delegation
+* Commission rates for node operators
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:12]:
```suggestion
* Commission management for node operators
```

File: src/operators/AmberNetwork [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* Consensus committee with up to 70 validators
+* Multiple Paratimes (parallel runtimes) running compute on
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:14]:
```suggestion
* Multiple ParaTimes (parallel runtimes) running compute on
```

File: src/operators/AmberNetwork [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+the network (expect more and more added over time)
+* Fast sync
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:16]:
```suggestion
```

File: src/operators/AmberNetwork [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+the network (expect more and more added over time)
+* Fast sync 
+
+##Timing
+
+The Amber Network will kick off in mid-June. We will share the specific start date on June 11. 
+
+## What’s the goal of this network?
+
+This network has been designed primarily for Node Operators
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:24]:
```suggestion
This network is intended primarily for node operators
```

File: src/operators/AmberNetwork [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+with tokens from the Questnet as a means to do a final test of the
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:25]:
```suggestion
who won tokens in The Quest as a means to do a final test of the
```

File: src/operators/AmberNetwork [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+Oasis Network code before the Mainnet launch. The network will also
+give very small grants to individuals who want to be wallet testers.
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:27]:
```suggestion
Oasis Network code before the Mainnet launch. Very small grants
will also be awarded to individuals who want to test hardware wallets.
```

File: src/operators/AmberNetwork [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+
+## Joining the Amber Network as a Node Operator
+
+In order to join the Amber Network, you will need AMBER test tokens.
+*(Note: AMBER tokens are test tokens used specifically for this network).* 
+Unlike the Quesnet, there is no faucet to receive tokens so instead you
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:33]:
```suggestion
Unlike in the QuestNet, there is no faucet to distribute tokens, so instead you
```

File: src/operators/AmberNetwork [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+can receive tokens in two ways:
+* **Genesis Block**: In the genesis block of the Network Node Operators
+will be given the total amount of tokens they’ve previously earned from
+the Oasis Questnet or via other means (fulfilled grants, etc). You will
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:37]:
```suggestion
* **Genesis Block**: In the genesis block of the Amber Network, node operators
will be allotted the total amount of tokens they’ve previously earned from
the Oasis QuestNet or via other means (fulfilled grants, etc). You will
```

File: src/operators/AmberNetwork [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+be able to find that amount here after the Questnet ends on June 4.
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:38]: Reminder: add link

Comment (peterjgilbert) [inline src/operators/AmberNetwork:38]:
```suggestion
be able to find that amount here after The Quest ends on June 4.
```

File: src/operators/AmberNetwork [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+* **Delegation** from the Oasis Foundation and Oasis Labs
+(primarily for testing purposes)
+
+Incentives for the Amber Network will be based on this community-proposed
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:42]: Reminder: add link

File: src/operators/AmberNetwork [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+incentive model and include a few parameters specific to this Network.
+
+## Adding your entity to the genesis block
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:45]:
```suggestion
## Submitting your entity for the genesis block
```

File: src/operators/AmberNetwork [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+The Amber Network will kick off in mid-June. We will share the specific start date on June 11. 
+
+## What’s the goal of this network?
+
+This network has been designed primarily for Node Operators
+with tokens from the Questnet as a means to do a final test of the
+Oasis Network code before the Mainnet launch. The network will also
+give very small grants to individuals who want to be wallet testers.
+
+## Joining the Amber Network as a Node Operator
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:29]:
```suggestion
## Joining the Amber Network as a node operator
```

File: src/operators/AmberNetwork [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+
+In order to join the Amber Network, you will need AMBER test tokens.
+*(Note: AMBER tokens are test tokens used specifically for this network).* 
+Unlike the Quesnet, there is no faucet to receive tokens so instead you
+can receive tokens in two ways:
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:34]:
```suggestion
can get stake needed to run your node in two ways:
```

File: src/operators/AmberNetwork [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+* **Delegation** from the Oasis Foundation and Oasis Labs
+(primarily for testing purposes)
+
+Incentives for the Amber Network will be based on this community-proposed
+incentive model and include a few parameters specific to this Network.
+
+## Adding your entity to the genesis block
+
+To ensure that your node is in the genesis block, we will ask that you
+submit a new entity package with fresh keys no later than Monday June 15.
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:48]:
```suggestion
submit a new entity package no later than Monday, June 15.
```

File: src/operators/AmberNetwork [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+Note that even if you’ve submitted an entity package in the past,
+we are asking for a new one now.
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:50]:
```suggestion
you need to regenerate and submit a new one.
```

File: src/operators/AmberNetwork [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+
+Entity package instructions can be found [here](docs.oasis.dev).
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:52]: Reminder: update link

File: src/operators/AmberNetwork [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+
+## Incentives
+
+The Amber Network will be used as a way to provide bonuses in the
+form of delegation from the Oasis Protocol Foundation. The Oasis Protocol
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:57]:
```suggestion
form of delegation from the Oasis Protocol Foundation in the forthcoming Mainnet. The Oasis Protocol
```

File: src/operators/AmberNetwork [hunk 21]
```diff
@@ -0,0 +1,72 @@
...
+
+In order to join the Amber Network, you will need AMBER test tokens.
+*(Note: AMBER tokens are test tokens used specifically for this network).* 
+Unlike the Quesnet, there is no faucet to receive tokens so instead you
+can receive tokens in two ways:
+* **Genesis Block**: In the genesis block of the Network Node Operators
+will be given the total amount of tokens they’ve previously earned from
+the Oasis Questnet or via other means (fulfilled grants, etc). You will
+be able to find that amount here after the Questnet ends on June 4.
+* **Delegation** from the Oasis Foundation and Oasis Labs
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:39]:
```suggestion
* **Delegation** from the Oasis Protocol Foundation and Oasis Labs
```

File: src/operators/AmberNetwork [hunk 22]
```diff
@@ -0,0 +1,72 @@
...
+Note that even if you’ve submitted an entity package in the past,
+we are asking for a new one now.
+
+Entity package instructions can be found [here](docs.oasis.dev).
+
+## Incentives
+
+The Amber Network will be used as a way to provide bonuses in the
+form of delegation from the Oasis Protocol Foundation. The Oasis Protocol
+Foundation will plan to allocate up to 50% of its available tokens based
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:58]:
```suggestion
Foundation will plan to delegate up to 50% of its available tokens based
```

File: src/operators/AmberNetwork [hunk 23]
```diff
@@ -0,0 +1,72 @@
...
+on success and staking reward rankings of the Oasis Amber Network as well
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:59]:
```suggestion
on successful completion and staking reward rankings from the Oasis Amber Network as well
```

File: src/operators/AmberNetwork [hunk 24]
```diff
@@ -0,0 +1,72 @@
...
+as availability rankings and overall track record. **(NOTE: Delegation is
+contingent on commission rates that are no more than +/- 10% of the 
+weighted median network commission rate)**
+
+## Wallet-Testers Needed
```
Comment (peterjgilbert) [inline src/operators/AmberNetwork:64]:
```suggestion
## Wallet testers needed
```

File: src/operators/amber-network.md [hunk 25]
```diff
@@ -0,0 +1,74 @@
...
+incentive model and include a few parameters specific to this Network.
+
+## Submitting your entity for the genesis block
+
+To ensure that your node is in the genesis block, we will ask that you
+submit a new entity package no later than Monday, June 15.
+Note that even if you’ve submitted an entity package in the past,
+you need to regenerate and submit a new one.
+
+Entity package instructions can be found [here](docs.oasis.dev).
```
Comment (peterjgilbert) [inline src/operators/amber-network.md:54]: TODO: update this link to point to entity package instructions

File: src/operators/amber-network.md [hunk 26]
```diff
@@ -0,0 +1,74 @@
...
+**(NOTE: Delegation is contingent on commission rates that are no more than +/-
+10% of the weighted median network commission rate)**
+
+## Wallet testers needed
+
+More on this to come soon!
+
+**Disclaimer: The Amber Network and all associated activities and rewards
+are not available to all participants and eligibility requirements apply.
+For details, see the Program Terms available here. This article does not
```
Comment (peterjgilbert) [inline src/operators/amber-network.md:72]: TODO: link to program terms

File: src/operators/amber-network.md [hunk 27]
```diff
@@ -0,0 +1,74 @@
...
+
+## Incentives for Joining
+
+The Amber Network will be used as a way to provide bonuses in the
+form of delegation from the Oasis Protocol Foundation in the forthcoming Mainnet.
+The Oasis Protocol Foundation will plan to delegate up to 50% of its available
+tokens based on successful completion and staking reward rankings from the Oasis
+Amber Network as well as availability rankings and overall track record.
+**(NOTE: Delegation terms to be determined with the
+Oasis Protocol Foundation at the end of the competition.)**
```
Comment (ravenac95) [inline src/operators/amber-network.md:37]:
```suggestion
::: tip NOTE: 
Delegation terms to be determined with the Oasis Protocol Foundation 
at the end of the competition.
:::
```

File: src/operators/amber-network.md [hunk 28]
```diff
@@ -0,0 +1,74 @@
...
+
+## Joining the Amber Network as a node operator
+
+In order to join the Amber Network, you will need AMBER test tokens.
+*(Note: AMBER tokens are test tokens used specifically for this network).*
```
Comment (ravenac95) [inline src/operators/amber-network.md:42]: I'd like to use the `::: tip NOTE` section but this is odd to break up.

File: src/operators/amber-network.md [hunk 29]
```diff
@@ -0,0 +1,74 @@
...
+
+## Wallet testers needed
+
+More on this to come soon!
+
+**Disclaimer: The Amber Network and all associated activities and rewards
+are not available to all participants and eligibility requirements apply.
+For details, see the Program Terms available [here](https://docsend.com/view/zv5cfia).
+This article does not guarantee anyone the right to participate in the
+Amber Network or to receive rewards.**
```
Comment (ravenac95) [inline src/operators/amber-network.md:74]:
```suggestion
::: warning DISCLAIMER 
The Amber Network and all associated activities and rewards
are not available to all participants and eligibility requirements apply.
For details, see the Program Terms available [here](https://docsend.com/view/zv5cfia).
This article does not guarantee anyone the right to participate in the
Amber Network or to receive rewards.
:::
```

File: src/operators/amber-network.md [hunk 30]
```diff
@@ -0,0 +1,74 @@
...
+The Oasis Amber Network is a release candidate for Mainnet and
+includes all the core features of the network you can expect
+at Mainnet launch.  This includes:
+
+* Fully decentralized consensus layer
+* Staking and delegation
+* Commission management for node operators
+* Consensus committee with up to 70 validators
+* Multiple ParaTimes (parallel runtimes) running compute on
+the network (expect more and more added over time)
```
Comment (ravenac95) [inline src/operators/amber-network.md:16]:
```suggestion
  the network (expect more and more added over time)
```

File: src/operators/amber-network.md [hunk 31]
```diff
@@ -0,0 +1,74 @@
...
+incentive model and include a few parameters specific to this Network.
+
+## Submitting your entity for the genesis block
+
+To ensure that your node is in the genesis block, we will ask that you
+submit a new entity package no later than Monday, June 15.
+Note that even if you’ve submitted an entity package in the past,
+you need to regenerate and submit a new one.
+
+Entity package instructions can be found [here](docs.oasis.dev).
```
Comment (ravenac95) [inline src/operators/amber-network.md:63]:
```suggestion
Entity package instructions can be found [here](./creating-an-entity-package.md).
```

File: src/operators/amber-network.md (hunk 25 above)
Comment (Afauvre) [reply src/operators/amber-network.md:63]: @ravenac95 can you add?

File: src/operators/amber-network.md (hunk 28 above)
Comment (Afauvre) [reply src/operators/amber-network.md:43]: I think that'd be okay.

File: src/operators/amber-network.md [hunk 32]
```diff
@@ -0,0 +1,80 @@
...
+## Incentives for Joining
+
+The Amber Network will be used as a way to provide bonuses in the
+form of delegation from the Oasis Protocol Foundation in the forthcoming Mainnet.
+The Oasis Protocol Foundation will plan to delegate up to 50% of its available
+tokens based on successful completion and staking reward rankings from the Oasis
+Amber Network as well as availability rankings and overall track record.
+**(NOTE: Initial delegation will be for at least 6 month and is contingent on
+commission rates that are +/- 10% of the weighted median network commission
+rate and no greater than 30% with high uptime performance.)**
```
Comment (peterjgilbert) [inline src/operators/amber-network.md:39]:
```suggestion
rate and no greater than 30%, and high uptime performance.)**
```

File: src/operators/amber-network.md [hunk 33]
```diff
@@ -0,0 +1,80 @@
...
+on June 11.
+
+## Goal of Amber
+
+This network is intended primarily for node operators
+who won tokens in The Quest as a means to do a final test of the
+Oasis Network code before the Mainnet launch. Very small grants
+will also be awarded to individuals who want to test hardware wallets.
+
+## Incentives for Joining
```
Comment (peterjgilbert) [inline src/operators/amber-network.md:30]:
```suggestion
## Incentives for joining
```

File: src/operators/amber-network.md [hunk 34]
```diff
@@ -0,0 +1,80 @@
...
+## Submitting your entity for the genesis block
+
+To ensure that your node is in the genesis block, we will ask that you
+submit a new entity package no later than Monday, June 15.
+Note that even if you’ve submitted an entity package in the past,
+you need to regenerate and submit a new one.
+
+Entity package instructions can be found [here](./creating-an-entity-package.md).
+
+## Wallet-testers needed
```
Comment (peterjgilbert) [inline src/operators/amber-network.md:67]:
```suggestion
## Wallet testers needed
```

File: src/operators/amber-network.md [hunk 35]
```diff
@@ -0,0 +1,80 @@
...
+
+In order to join the Amber Network, you will need AMBER test tokens.
+::: Tip *Note: AMBER tokens are test tokens used specifically for this network).*
+Unlike in the QuestNet, there is no faucet to distribute tokens, so instead you
+can get stake needed to run your node in two ways:
+
+* **Genesis Block**: In the genesis block of the Amber Network, node operators
+will be allotted the total amount of tokens they’ve previously earned from
+the Oasis QuestNet or via other means (fulfilled grants, etc). You will
+be able to find that amount here after The Quest ends on June 4.
```
Comment (peterjgilbert) [inline src/operators/amber-network.md:51]: Needs a link to quest winners?

File: src/operators/amber-network.md [hunk 36]
```diff
@@ -0,0 +1,80 @@
...
+will also be awarded to individuals who want to test hardware wallets.
+
+## Incentives for joining
+
+The Amber Network will be used as a way to provide bonuses in the
+form of delegation from the Oasis Protocol Foundation in the forthcoming Mainnet.
+The Oasis Protocol Foundation will plan to delegate up to 50% of its available
+tokens based on successful completion and staking reward rankings from the Oasis
+Amber Network as well as availability rankings and overall track record.
+**(NOTE: Initial delegation will be for at least 6 month and is contingent on
```
Comment (peterjgilbert) [inline src/operators/amber-network.md:37]:
```suggestion
**(NOTE: Initial delegation will be for at least 6 months and is contingent on
```

**Pull request merged**